contrib/ray/raycluster_example.yaml

```diff
@@ -1,60 +1,96 @@
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: allow-ray-workers-head
+spec:
+  action: ALLOW
+  rules:
+  - from:
+    - source:
+        principals:
+        - "cluster.local/ns/kubeflow-user-example-com/sa/kubeflow-raycluster"
```

Comment:
This must be generic. The namespace could also have another name that you do not know in advance. The same manifests should work for different users.

Reply:
Is there a way to templating that namespace value without using helm?

Reply:
The Serviceaccount is wrong. It should be default-editor everywhere. Especially in the raycluster itself. 

You can take a look at https://istio.io/latest/docs/reference/config/security/authorization-policy/ for ideas and ask gpt4. Maybe you can just whitelist the whole namespace.

Reply:
Is it only enough to generalize the service account name(with default-editor)? Because since we are deploying it in a namespace, we have to specify the namespace to give the access to only for that specific namespace(kubeflow-user-example in this case)

Reply:
No the other way around. We need to generalize the namespace. It would be alright to allow all serviceaccounts or all pods from the namespace. But the deployments should still run as default-editor. This must be deployable without manual changes per namespace.

Reply:
Are you setting the default-editor in the raycluster resource itself?

Reply:
I added the default-editor service account to the deployment. 
> We need to generalize the namespace.

This means should I add like this,
cluster.local/ns/kubeflow-user-example-com/sa/*

Reply:
I did the changes . @juliusvonkohout